source/simulator/parameters.cc

```diff
@@ -657,6 +658,40 @@ namespace aspect
+                           "the discontinous temperature solution. Currently we apply this only to the"
+                           "temperature solution if the global max/min are already determined and defined"
+                           "This limiter keeps the discontinuous solution in the range given by"
+                           "Global temperature max or min.");
+        prm.declare_entry ("Use limiter for discontinuous composition solution", "false",
+                           Patterns::Bool (),
+                           "Whether to apply the bound preserving limiter as a correction after having"
+                           "the discontinous composition solution. Currently we apply this only to the"
+                           "compositional solution if the global max/min are already determined and defined"
+                           "This limiter keeps the discontinuous solution in the range given by"
+                           "Global composition max or min.");
+        prm.declare_entry ("Global temperature max", boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
+                           Patterns::Double (),
+                           "The maximum global temperature value that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from temperature advection fields."
+                           "Usually it's predetermined by the initial temperature field");
+        prm.declare_entry ("Global temperature min", boost::lexical_cast<std::string>(-std::numeric_limits<double>::max()),
+                           Patterns::Double (),
+                           "The minimum global temperature value that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from temperature advection fields."
+                           "Usually it's predetermined by the initial temperature field");
+        prm.declare_entry ("Global composition max", boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
+                           Patterns::List(Patterns::Double ()),
+                           "The maximum global composition values that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from composition advection fields."
+                           "Usually it's predetermined by the initial compositional field");
+        prm.declare_entry ("Global composition min", boost::lexical_cast<std::string>(-std::numeric_limits<double>::max()),
+                           Patterns::List(Patterns::Double ()),
+                           "The minimum global composition value that will be used in the bound preseving"
+                           "limiter for the discontinuous solutions from composition advection fields."
+                           "Usually it's predetermined by the initial compositional field");
```

Comment:
Right now the correct global max/min values have to be defined by the user in the .prm file. It's the same as the maximum/minimum of the initial filed for a pure advection equation without other source term, however it's usually expensive to find the maximum/minimum numerically ?... So I only leave a note without any implementation.

Perhaps I should remove the last sentence?